app/controllers/idv/in_person/usps_locations_controller.rb

```diff
@@ -0,0 +1,57 @@
+require 'json'
+
+module Idv
+  module InPerson
+    class UspsLocationsController < ApplicationController
```

Comment:
(Non-blocking) We have some precedent for [namespacing](https://github.com/18F/identity-idp/tree/main/app/controllers/api) or [naming routes](https://github.com/18F/identity-idp/blob/b25790b34d3c542f00849808dc37e8d0a7ec64e7/config/routes.rb#L343-L348) as "API" when they talk exclusively JSON. Should we do that here?

Reply:
For clarification - is the suggestion here that the controller should be renamed to "UspsLocationsAPIController"?

Reply:
I was thinking in the module namespace, so e.g. `Api::Verify::InPerson::UspsLocationsController`. This is similar to what's done elsewhere.

```rb
module Api
  module Verify
    module InPerson
      class UspsLocationsController
```

And then we'd have the route be something like `/api/verify/in_person/usps_locations`.

Reply:
oh ok , got it!